client/layout/logged-out.jsx

```diff
@@ -3,7 +3,8 @@ import { localizeUrl } from '@automattic/i18n-utils';
 import classNames from 'classnames';
```

Comment:
Change was added to both default and logged out layouts instead of adding into SSR or normal loaded app versions.